src/components/Nav/BackgroundDropdownItem.tsx

```diff
@@ -19,9 +22,38 @@ export const BackgroundDropdownItem = ({
         "cursor-pointer bg-gray-300 py-1 hover:bg-gray-100 dark:bg-gray-800 dark:hover:bg-gray-900",
         isPicked && "bg-gray-100 dark:bg-gray-900"
       )}
-      onClick={() => setIsBackground(background)}
+      onClick={() => setBackgroundId(background)}
     >
-      <div className="block px-4 py-2 text-sm text-gray-700 dark:text-gray-200">{title}</div>
+      {title === "Custom Color" ? (
+        <TestColorPicker />
+      ) : (
+        <div className="block px-4 py-2 text-sm text-gray-700 dark:text-gray-200">{title}</div>
+      )}
+    </div>
+  );
+};
+
+const colors = ["#000000", "#383838", "#654724", "#312465", "#1F6353", "#652424"];
```

Comment:
Would it be more ideal to have an enum of background colors stored for example in `interfaces.ts` for reusability purposes?

Reply:
It would be do this. I'm not sure if anything else will use the color list, but it would be good practice.

Reply:
if we used this elsewhere this would be a good idea otherwise here it's more of a nice to have.